src/Driver/Http1Driver.php

```diff
@@ -475,25 +479,25 @@ private function parser(): \Generator
                     );
                 }
 
-                if (isset($headers["expect"][0]) && \strtolower($headers["expect"][0]) === "100-continue") {
+                if (isset($headerMap["expect"][0]) && \strtolower($headerMap["expect"][0]) === "100-continue") {
                     yield $this->write(
-                        new Request($this->client, $method, $uri, $headers, null, $protocol),
+                        new Request($this->client, $method, $uri, $headerMap, null, $protocol),
```

Comment:
Shouldn't this also use the `$headers` variables and add the headers manually?

Reply:
This `Request` instance is only used internally, it is never given to a `RequestHandler`, so it doesn't matter since the original header case is never used.